examples/deploy/ocm-token/install-csr-token.sh

```
@@ -0,0 +1,31 @@
+
+export PATH=$PATH:${WORK_DIR}/bin
+
+echo "############  Install clusteradm"
+go install open-cluster-management.io/clusteradm/cmd/clusteradm@main
+
+echo "############ Init hub"
+${CLUSTERADM} init --wait --bundle-version=latest
+joincmd=$(${CLUSTERADM} get token | grep clusteradm)
+
+echo "############ Init agent as cluster1"
+$(echo ${joincmd} --force-internal-endpoint-lookup --wait --bundle-version=latest --addon-kubeclient-registration-auth=token | sed "s/<cluster_name>/${MANAGED_CLUSTER_NAME}/g")
```

Shellcheck (0.11.0)

[warning] Remove surrounding $() to avoid executing output (or use eval if intentional).

(SC2091)